lib/fog/openstack/models/collection.rb

```diff
@@ -34,7 +34,10 @@ def summary(options = {})
       def get(uuid)
         raise Fog::OpenStack::Errors::InterfaceNotImplemented.new('Method :get is not implemented')
       end
-      alias_method :find_by_id, :get
+
+      def find_by_id(uuid)
+        get(uuid)
+      end
```

Comment:
gah, seems like this bug was here quite a while. Alias works weird with inheritance, so it always points to the above method, not the one implemented in the subclass. The things we discover with model tests. :-)